### PR TITLE
Fix generating kernel command line for buildiso netiso

### PR DESCRIPTION
**Pull request by aaannz**
Fixes https://github.com/SUSE/spacewalk/issues/20331

### CODE REVIEW

**Review by meaksh — APPROVED** [review]
Looks good to me :+1:

It seems this issue is also affecting upstream Cobbler, could you please create upstream PR?

Thanks!

**Comment by aaannz** [comment]
Created ports for `uyuni/master`, `suma/head` and `cobbler/main`. Did I miss anything else? I can't merge the PRs after review, so if you would be so kind to merge them.

**Pull request merged**
